### PR TITLE
Added a 'water-mob-protection' option that makes the water block that spawns when water mobs spawn toggleable.

### PR DESCRIPTION
**Pull request by OliPulse**

### CODE REVIEW

**Comment by maxlehot1234** [comment]
@tastybento Check this :) OliPulse was my dev. I test this thing and it works perfectly on my server ;)

File: src/main/java/world/bentobox/aoneblock/listeners/BlockListener.java [hunk 1]
```diff
@@ -343,7 +343,11 @@ private void makeSpace(Entity e) {
                 for (; y <= Math.min(bb.getMaxY() + 1, world.getMaxHeight()); y++) {
                     b = world.getBlockAt(new Location(world, x,y,z));
                     if (!b.getType().equals(Material.AIR) && !b.isLiquid()) b.breakNaturally();
-                    b.setType(WATER_ENTITIES.contains(e.getType()) ? Material.WATER : Material.AIR, false);
+                    if (addon.getSettings().isWaterMobProtection()) {
+                        b.setType(WATER_ENTITIES.contains(e.getType()) ? Material.WATER : Material.AIR, false);
```
Comment (tastybento) [inline src/main/java/world/bentobox/aoneblock/listeners/BlockListener.java:347]: Replace lines 346 to 350 with this one line:

```
b.setType(WATER_ENTITIES.contains(e.getType()) && addon.getSettings().isWaterMobProtection() ? Material.WATER : Material.AIR, false);
```

Comment (OliPulse) [reply src/main/java/world/bentobox/aoneblock/listeners/BlockListener.java]: This indeed is better. I changed it, thanks.

**Comment by tastybento** [comment]
Thanks!

**Pull request merged**